.changes/v2.15.0/454-features.md

```diff
@@ -0,0 +1,3 @@
+* Added support to set, get and delete metadata to any supported resource via its HREF, with the methods
```

Comment:
What about adding a list or just an explanation about which resources support this generic metadata function and which don't?

Reply:
Done, please check if you're OK with the formatting or I can refine it.

Reply:
Looks good to me, @vbauzysvmware could you review the list and the way we provide resource names too? E.g. is "disk" specific enough, etc. Thanks.

Reply:
(Just for context, picked namings from [here](https://developer.vmware.com/apis/1046/vmware-cloud-director/doc/doc/landing-user_operations.html))